app/models/user.rb

```diff
@@ -47,13 +47,24 @@ def self.find_or_create_from_gov_one(email:, gov_one_id:)
     user
   end
 
+  # @return [User]
+  def self.test_user
+    find_or_create_from_gov_one(
+      email: 'completed@example.com',
+      gov_one_id: 'urn:fdc:gov.uk:2022',
```

Comment:
If we only want this in review apps, we should already have seeds, so we just need to find.

Reply:
Can we get a test in place for this

Reply:
[0f11516](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/1078/commits/0f115161c8d8682f0a602695c295cf2eaa062ccf)